app/lib/rocketchat.js

```diff
@@ -1032,7 +1032,27 @@ const RocketChat = {
 		// RC 0.42.0
 		return this.methodCallWrapper('getUsersOfRoom', rid, allUsers, { skip, limit });
 	},
+	getMembers(rid, roomType, type, debouncedText, start = 0, end = 10) {
```

Comment:
There's no written rule for that, but there's enough params to be an object instead :)
Also, I didn't get why we gave `offset/count` a new naming.

Reply:
No reason actually 🤔 Was using the web implementation as reference, I'll change that :)